src/pages/ChatFinderPage/index.tsx

```diff
@@ -120,14 +120,17 @@ function ChatFinderPage({betas, isSearchingForReports, navigation}: ChatFinderPa
 
         if (recentReports?.length > 0) {
             newSections.push({
-                data: recentReports.map((report) => ({...report, isBold: report.isUnread})),
+                data: recentReports.map((report) => {
+                    const isBold = report.isUnread && report.notificationPreference !== CONST.REPORT.NOTIFICATION_PREFERENCE.MUTE;
```

Comment:
Maintaining same logic in difference places can cause future inconsistency issues. Can we move this to a util function?